### ISSUE TITLE
Plans to support alerts based on log lines?

### ISSUE THREAD

**Issue by tboerger** [flant/loghouse#83]
Hi,

are there any plans to support alerts based on log lines and maybe some search pattern?

That would be really useful :)

**Comment by diafour** [comment]
#74 is exactly your case.

**Comment by tboerger** [comment]
> #74 is exactly your case.

Not directly, since prometheus can only use numeric values you can get a bare alert without any useful context or any kind of url to a loghouse query to get more information. So IMHO this is a more or less different use case.

**Comment by diafour** [comment]
Url can be passed as label as a workaround. Nevertheless alerting should be done differently.

Alerting can be implemented like this after resolution #74
```
Clickhouse
↓  result of the query
Loghuose
↓  values of metrics
Prometheus
↓  query prometheus db, check for threshold values, send alert message
Alertmanager
↓  send alerts
```

This process is complicated. We can implement alerting and metrics independently of one another. Alerting process will be like this:
```
Clickhouse
↓  result of the query
Loghouse
↓  check for threshold value, send alert message
Alertmanager
↓  send alerts
```

This should be faster and users will not have "only numeric values" restriction.

@may-cat @qw1mb0 @gsmetal FYI

**Comment by tboerger** [comment]
Closing this since I'm not using loghouse anymore.
